libtenzir/builtins/connectors/tcp.cpp

```diff
@@ -782,8 +791,58 @@ class plugin final : public virtual loader_plugin<loader>, saver_plugin<saver> {
   }
 };
 
+class save_tcp final : public virtual operator_plugin2<saver_adapter<saver>> {
+  auto
+  make(invocation inv, session ctx) const -> failure_or<operator_ptr> override {
+    // TODO: Do we want the `listen` and `listen_once` options?
```

Comment:
I don't think we need them for now. Let's wait until someone asks for them.

Reply:
Should we keep the commented lines?

Reply:
Don't think so.